app/javascript/src/Onboarding.jsx

```diff
@@ -182,8 +182,7 @@ class Onboarding extends Component {
           this.setState(prevState => ({
             allTags: prevState.allTags.map(currentTag => {
               const newTag = currentTag;
-              newTag.following =
-                currentTag.name === tag.name ? json.outcome : 'followed';
+              newTag.following = json.outcome === 'followed';
```

Comment:
This part changed the functionality. You need to still check if  `currentTag.name === tag.name`. I'll change this for you.